.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -7,22 +7,33 @@ assignees: ''
 ---
 <!-- Hey. Our contribution guidelines (https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) might be an appropriate
  document to read before you fill out the request :) -->
-#### Is your feature request related to a problem? Please describe it
```

Comment:
Related issues and PRs should be mentioned somewhere

Reply:
Ah. Point.

Reply:
Okay now?